05-how-to-contribute-to-open-source.md

```diff
@@ -16,165 +20,329 @@ OpenSauced is a powerful tool for finding open source projects to contribute to.
 
 By following these steps, you'll be able to submit your contributions to open source projects and collaborate with other developers to improve the codebase.
 
+## What Happens Next?
+
+After your contribution has been submitted and reviewed, one of the following outcomes may occur:
+
+1. **Your contribution is accepted**: If your contribution is approved by the project maintainers, it will be merged into the main branch of the codebase. <br>
+   Congratulations! Your work is now part of the project, and you've made a valuable contribution to the open source community.
+
+2. **Your contribution requires changes**: Sometimes, the project maintainers may request changes to your contribution before it can be accepted. This could be due to coding issues, conflicts with other changes, or a need for additional documentation. In this case, make the requested changes and resubmit your pull request.
+
+3. **Your contribution is rejected**: In some cases, your contribution may not align with the project's goals or requirements, or it may not be the best solution to a problem. If your contribution is rejected, don't be discouraged. Take the feedback you received as an opportunity to learn and improve. You can always try contributing to another project or submitting a different contribution to the same project.
+
 ## Let's Get Practical
 
-Now that you know how to find and contribute to open source projects, it's time to put your skills into practice. Let's do this by contributing to OpenSauced's guest book using the [all contributors specification](https://allcontributors.org/).
+Now that you know how to find and contribute to open source projects, it's time to put your skills into practice. Let's do this by contributing to OpenSauced's [guest book repository](https://github.com/open-sauced/guestbook).
+
+### Prerequisite
+
+You'll need to have these tools donwloaded and installed on your local machine:
+
+- [Node.js](https://nodejs.org)
+- [Visual Studio Code (VS Code)](https://code.visualstudio.com/)
+
+### Getting Started
+
+1. Fork the [guest book repository](https://github.com/open-sauced/guestbook).
+2. Clone your forked repository to your computer.
+3. Run `npm install` to install the dependencies.
+4. Create a new branch and use a descriptive name related to your contribution — for example, `feat/add-alice`.
+5. Add yourself to the guest book.
+
+   There are two ways you can complete this activity:
+
+   - Using the CLI tool.
+   - Manually editing the `.all-contributorsrc` file.
+
+   Although adding yourself to the guest book with the CLI tool is simpler, we recommend manually editing the file to get more experience adding and committing your changes, creating pull requests, and pull request reviews. Below, you'll find the directions for each method:
```

Comment:
This is just a note to say that we should evaluate this part in the future. We may want to encourage the CLI tool and then say something like, Once you've contributed to the guestbook, you can get more practice with the Pizza-verse repo or something. I think it would reduce the number of incomplete PRs.

Reply:
We definitely can review it again 😊

What I see often happen is that folks tend to miss the step to generate the README either they use CLI or add manually.

The concern with adding manually is that if they don't stick to the list of there's a typo, it will break the CLI.